react/fields/Dropdown/Dropdown.js

```diff
@@ -152,14 +156,12 @@ export default class Dropdown extends Component {
           { placeholder }
         </option>
         {
-          options.map(({ value, label }) => (
-            <option
-              value={value}
-              key={value}
-              className={styles.option}>
-              { label }
-            </option>
-          ))
+          options.map(({ value, label }) => {
+            if (Array.isArray(value)) {
+              return (<optgroup value="" label={label} key={label}>{value.map(this.renderOption)}</optgroup>);
```

Comment:
Do we have any default styling for our option group names drop downs? @tcrook1 @stephaniemoss?